### ISSUE TITLE
What's the recommended way to use this with the frontend?

### ISSUE THREAD

**Issue by abienz** [boxing/boxrec#243]
Hi, I've read through your examples and a few issues on here, to try and work out what the best way to use this API is, but it seems that maybe I missunderstand its usecase.

Specifically, as per issue https://github.com/boxing/boxrec/issues/65 I was hoping to get a JSON object passed to my frontend via a fetch call, so that I could update my client page with boxer data.

According in the answer in that post, the feature of a JSON object being available on a boxrec.getPersonById call has been disabled, is that right?

Instead we have to use getters `boxer.name` for each piece of data. 

Wouldn't this mean I would have to build my own JSON object in the backend before sending it to the front end? Or I would be in danger of making many requests to the API which wouldn't be efficient.

Do you have suggestions for minimal code to send through boxers data from an express node backend to a simple html(template) and js frontend based on some AJAX call?

**Comment by mikedidomizio** [comment]
If I understand correctly, you could use
```
const boxerData= await boxrec.getPersonById(352)
const boxerOutput = boxerData.output;

```

If you `console.log(boxerOutput)`, you should see an object literal structure that includes name, date of birth, bouts etc.

I was also in the process of building an express/node (actually it's NestJS) dockerized backend but never really got far into it.  There should be some working examples although the project may have become a bit deprecated in the meantime.

You can see an example of doing a [profile request here](https://github.com/boxing/boxrec-docker/blob/master/src/pages/profile/profile.controller.ts)

Let me know if that helps

**Comment by abienz** [comment]
Yes, that's what I was looking for, thank you, I noticed this isn't mentioned on the front page, is there a better source for the API I should be looking at?

**Comment by mikedidomizio** [comment]
So when everything is merged into `master` and a release is made the documentation should update itself.  That being said, the `README` is manually updated and I've been the only one currently maintaining it unfortunately.  It doesn't mention anything about the `output` keys or how to use it efficiently.

It's a todo for sure.  I'll keep this issue open until I or someone else makes some decent updates to it
